met-web/src/components/landing/EngagementTile.tsx

```diff
@@ -76,78 +80,104 @@ const EngagementTile = ({ passedEngagement, engagementId }: EngagementTileProps)
-                        </Button>
-                    </Then>
-                </If>
-                <If
-                    condition={
-                        submission_status === SubmissionStatus.Closed || submission_status === SubmissionStatus.Upcoming
-                    }
+        <Link to={engagementUrl} sx={{ textDecoration: 'none' }} tabIndex={-1}>
+            <ThemeProvider theme={isHovered || isFocused ? DarkTheme : BaseTheme}>
+                <Card
+                    tabIndex={0}
+                    onMouseEnter={() => setIsHovered(true)}
+                    onMouseLeave={() => {
+                        setIsHovered(false);
+                        setIsActive(false);
+                    }}
+                    onFocus={() => setIsFocused(true)}
+                    onBlur={() => setIsFocused(false)}
+                    onMouseDown={() => setIsActive(true)}
+                    onMouseUp={() => setIsActive(false)}
+                    className={isActive ? 'active' : ''}
+                    sx={{
+                        borderRadius: '24px',
+                        width: '320px',
+                        cursor: 'pointer',
+                        '&:hover': {
+                            boxShadow: elevations.hover,
+                            background: colors.surface.blue[90],
+                        },
+                        '&.active': {
+                            boxShadow: elevations.pressed,
+                            background: colors.surface.blue[100],
+                        },
+                        '&:focus': {
+                            boxShadow: elevations.hover,
+                            background: colors.surface.blue[90],
+                        },
+                        '&:focus-visible': {
+                            boxShadow: globalFocusVisible + elevations.hover,
+                            background: colors.surface.blue[90],
+                        },
+                    }}
                 >
-                    <Then>
-                        <Button variant="secondary" fullWidth onClick={handleClick as () => void}>
-                            {translate('buttonText.viewEngagement')}
-                        </Button>
-                    </Then>
-                </If>
-            </CardActions>
-        </Card>
+                    <CardActionArea tabIndex={-1}>
+                        {Boolean(banner_url) && <CardMedia sx={{ height: '172px' }} image={banner_url} title={name} />}
+                        <CardContent sx={{ height: '260px', p: '40px 32px' }}>
+                            <Box
+                                sx={{
+                                    height: '96px',
+                                    mb: '32px',
+                                    width: '100%',
+                                    display: 'flex',
+                                }}
+                            >
+                                <EyebrowText
```

Comment:
Can I ask what your intention is with this component? Is it just meant to be a general text tag? If another developer wanted to use it, how would we know when to use it as oppose to a regular span or paragraph tag?